src/services/findAllReferences.ts

```diff
@@ -648,9 +648,11 @@ namespace ts.FindAllReferences.Core {
         }
 
         // If the symbol has a parent, it's globally visible.
+        // (Private properties handled above.)

```

Comment:
I just figured if someone read "If the symbol has a parent, it's globally visible.", they would try thinking of an exception -- want me to remove? Or I could just reword it to "If a non-private symbol has a parent, it's globally visible, unless it's a type parameter."